server/serverAPI/model/ingredient/requestSchema/InventoryIngredientSchema.ts

```diff
@@ -1,20 +1,19 @@
-import { IsNumber, IsPositive } from "class-validator";
 import IInventoryIngredient from "../IInventoryIngredient";
 import BaseIngredientSchema from "./BaseIngredientSchema";
 
+import IsType from "../../../../utils/ClassValidator";
```

Comment:
Not in this PR, but consider using one of the approaches from [this article](https://marian-caikovski.medium.com/how-to-import-a-module-from-the-root-of-a-node-js-project-edb3421534e3) to clean up your imports